src/main.rs

```diff
@@ -149,13 +149,25 @@ struct Options {
     /// Utility log level
     ///
     /// Set to `debug` when submitting a bug report.
+    /// Set to `info` to only print useful information.
+    /// Set to `Warn` to only print on hazardous situations.
+    /// Set to `Error` to only print serious errors.
```

Comment:
these two should be all-lowercase `warn` and `error`

Also the formatting for this in the help page isn't going to be great, it will show up as

```
Set to `debug` when submitting a bug report. Set to `info` to only print useful
information. Set to `warn` to only print on hazardous situations. Set to `error`
to only print serious errors.
```

Could instead use clap's possible_values (or whatever it's called) which should add a concise listing to the help.

Reply:
I've used `possible_values`, though it does not list the help I added, so I also formatted the doc to make it readable.